lib/graphql/relay/mutation.rb

```diff
@@ -199,6 +199,11 @@ def build_result(mutation_result, args, ctx)
           end
 
           if @wrap_result
+            unless mutation_result == nil || mutation_result.is_a?(Hash)
```

Comment:
Haha, I'm in the same boat, in my head I translate `unless` to "if not" every time. This could even be

```ruby
if !mutation_result.is_a?(Hash)
```

since everything responds to `is_a?`.